KsApi/models/graphql/adapters/Project+ProjectFragment.swift

```diff
@@ -83,21 +84,14 @@ private func finalCollectionDateTimeInterval(
 }
 
 /**
- Returns a minimal `Project.Stats` from a `ProjectFragment`
+ Returns a minimal `Project.MemberData` from a `ProjectFragment`
  */
-private func projectStats(from projectFragment: GraphAPI.ProjectFragment) -> Project.Stats {
-  return Project.Stats(
-    backersCount: projectFragment.backersCount,
-    commentsCount: nil,
-    convertedPledgedAmount: nil,
-    currency: projectFragment.currency.rawValue,
-    currentCurrency: nil,
-    currentCurrencyRate: nil,
-    goal: projectFragment.goal?.fragments.moneyFragment.amount.flatMap(Int.init) ?? 0,
-    pledged: projectFragment.pledged.fragments.moneyFragment.amount.flatMap(Int.init) ?? 0,
-    staticUsdRate: projectFragment.usdExchangeRate.flatMap(Float.init) ?? 0,
-    updatesCount: nil
-  )
+private func projectMemberData(from projectFragment: GraphAPI.ProjectFragment) -> Project.MemberData? {
```

Comment:
Assuming we get some backend support to add the other three properties of `MemberData`, we would extend this method to support that.